app/helpers/admin/users_helper.rb

```diff
@@ -21,5 +21,15 @@ def cascading_high_level_roles(user)
         "Resource Admin"
       end
     end
+
+    def format_role_tooltip(user)
+      if user.super_admin?
+        "Super Admin"
+      elsif user.admin?
+        "Admin"
+      elsif user.single_resource_admin_for?(:any)
+        "Resource Admin: #{user.roles.pluck(:resource_type).join(', ')}"
```

Comment:
Just one small thing I noticed - if a user has the "Trusted" role, as well as resource admin roles, we seem to include "Trusted" in the list, but as an empty string... so it ends up looking like we have a rogue comma -
![resources text with a comma at the beginning](https://user-images.githubusercontent.com/20773163/162401011-0eac634c-9398-420f-89b4-8c3cf3053ef6.png)

In this example above, the roles added to the member are Trusted, Resource admin: BadgeAchievement, Resource admin: DataUpdateScript

Reply:
Great catch! I'll work on removing "Trusted".